server/public/styles.css

```diff
@@ -1,4 +1,21 @@
 
 header{
-  padding:20px;
+  margin-top: 15px;
+}
+
+.header-row{
+  display:flex;
+  justify-content: space-between;
+}
```

Comment:
Bootstrap includes helper for flex and spacing so you can usually avoid custom CSS for these.

https://getbootstrap.com/docs/4.4/utilities/flex/
https://getbootstrap.com/docs/4.4/utilities/spacing/